src/locutus_util/dispatcher.py

```diff
@@ -1,27 +1,30 @@
 import argparse
 from locutus_util.ontology_api_etl import ontology_api_etl
 from locutus_util.seed_data_etl import seed_data_etl
+from locutus_util.delete_project_data import delete_project_data
 from locutus_util.common import (
     FETCH_AND_UPLOAD, UPLOAD_FROM_CSV, UPDATE_CSV,
-    UPDATE_ONTOLOGY_API, UPDATE_SEED_DATA
+    UPDATE_ONTOLOGY_API, UPDATE_SEED_DATA, DELETE_PROJECT_DATA,RESET_DATABASE
 )
 
 def main():
     parser = argparse.ArgumentParser(
         description="Manage data ETL of data into Firestore."
     )
     parser.add_argument(
-        '-p', '--project', 
+        '-p', '--project_id', 
```

Comment:
This isn't hugely important, but there is a common convention where linux CLI arguments use kebab-case rather than underscores.